### ISSUE TITLE
Permissions - Icewindo

### ISSUE THREAD

**Issue by NonovUrbizniz** [CiFor/DayZ_Sahrani#75]
- 1st Exchange:

[QUOTE=Icewindo][QUOTE=CiforDayZServer][QUOTE=Icewindo]

Hi sure you can use the models just no commercial BS like dayz orig allowed. Thanks for asking and good luck for your project - ive  player arma1 for a while so i know the map pretty well :).[/QUOTE]

Well, you wished it, we got it! Bohemia gave us the source files for the Sahrani Map, and we're being allowed to use the entire sample model library to retexture the assets for a FULL Arma 2 port of Sahrani!!!!

AND they are letting us build interiors for anything in the sample model library in order to open the map up for better play on DayZ.

Here's a WIP progress thread if you're interested:

[url]http://forums.bistudio.com/showthread.php?155076-WIP-Sahrani-Arma2-Port-Arma-1-Buildings-Project[/url]

Our 1st priority is doing a 1 for 1 port to ensure Arma guys can use the map 100pct for their already existing missions, we're getting advised on the project by .kju and bushlurker! We're very excited.

If you have ANY interest in helping or if you want to follow the progress of either or both projects please not hesitate to ask!

Q's for you!
1. Once released we will likely point to your Sahrani Citizens, and Wld427's Project RACS, as they are both so perfect for eachother. Just wanted to be sure you were ok with that firstly (it's a way off still obviously, we are just now in the process of beginning the map editing, and have not begun the retexture work on the assets as yet. Still waiting on some files from BI too). If you want we might be able to arrange a temporary mirror to offset some of the bandwidth if that is a concern.
2. May we not only use, but edit your Sahrani citizens? I saw in the original thread they had at some point been skinned for the other Zombie mod that came before DayZ. We would LOVE to retexture them in Zombie style for our DayZ project.
3. Would you be interested (long term goal here we have no thoughts of distracting ourselves with A3 until the FULL A2 port AND our customized version of Sahrani are DONE for DayZ) in doing the Sahrani Citizens for A3?

The main reason we are not at all focused on A3 is obviously our personal interest is in DayZ. As I understand it (trust it will be clarified on his return) Dean Hall DOES NOT want DayZ modders to port it to A3. So if that is the case we will not do that, I frankly feel weird that he's never directly given me permission, but one of our guys is on the DayZ Mod Dev team now, and has a "nod" from R4Z0R, which I suppose sort of counts... but I'd feel much better if I got those magic 2 letters from Rocket "OK".

HOWEVER when we do complete our own DayZ A2 project we would LOVE to port the map fully for A3 use regardless of the outcome of DayZ use. I am an avid SCUBA diver and would LOVE to redo Sahrani with SCUBA in mind as far as sea floor topography and landscape... So no matter what we will eventually be bringing Sahrani to A3.

Thanks for your time, Sorry for the novel!

Mike Dillon.[/QUOTE]

Hi,

sure you can use the Sahrani Civs Port and edit it, the source files should be included somewhere on ArmAholic or in the thread afaik, if it's not there I'll send you the files. I lack the time to help you actively though, so all the best with your project :) .

Icewindo[/QUOTE]
- 2nd Exchange:

[QUOTE=Icewindo]Hi Mike,

[QUOTE=CiforDayZServer]Hello again!

Further to our previous exchanges, I have put together the below list of ALL the addons we hope to use of yours. Just let me know: NO we can't use, Yes but ONLY use, and Yes and mod if we like. 

Thanks for your time and sorry for the long list, you're very talented and we have huge plans for the mod long term.

Hazmat Suits:
[url]http://forums.bistudio.com/showthread.php?143892-Hazmat-Units-(OA)[/url]
[B]Use OK[/B]

PMC Lite (Units by Schnapsdrosel)
[url]http://forums.bistudio.com/showthread.php?112498-PMC-Character-Weapon-Replacement-CO[/url]
[B]Do you you mean using the units? Or the config addon? I can only give permission for the config addon, not the units themselves, but I don't think schnapsdrossel wouldn't mind as long as the units are unmodified[/B]

Ballistic Shield (per your thread I'm sure our guys could script the bullet specific damage and release it back to you):
[url]http://forums.bistudio.com/showthread.php?87313-Ballistic-Shield[/url]
[B]Use OK - the bullet dmg is suffering from an engine limitation/bug though that still exists in A3. The bug is that once any fire geometry of a certain weapon class, e.g. my shield is destroyed, it is also destroyed for all duplicates of the same class existing currently ingame (which sucks big time and its sad BIS didnt fix it yet, dunno if a bug ticket exists)[/B]

Press (Obviously it would be awesome to have the source for this as we'd love to do an aged version of Mirian):
[url]http://forums.bistudio.com/showthread.php?94960-Faux-News-Reporters[/url]
[B]Use OK - here are the sources: [url]http://www.mediafire.com/?cglpbl9pbxboi8g[/url] [/B]

Paramilitary Units:
[url]http://forums.bistudio.com/showthread.php?96245-Paramilitary-Units[/url]
[B]Use OK - these were just reskins of normal A2 units[/B]

Saw this thread and believe it turned out that it was in the game already? But do you have any info on that? we're doing bricks on our mod. 

[url]http://forums.bistudio.com/showthread.php?82965-Throwable-stones-for-AI[/url]
[B]Yeah the stones are allready ingame, they use class HandGrenade_Stone or sth. like that. I would ask Icebreaker (who did the panthera islands and more...), afaik he was using a modified version of the stones that were more accurate in one of his packs as the vanilla stones are not really accurate[/B]

Also we MIGHT be interested in the Stalker units but that is under share share alike anyway right?
[url]http://forums.bistudio.com/showthread.php?96359-Stalker-Units-0-5a[/url]
[B]Yep Use OK[/B]

Thanks for your time, and again really, thank you for all your contribution to the community over the years!

[B]Thanks and all the best for your project :)[/B]

Mike Dillon[/QUOTE][/QUOTE]
